jade/distribution.yaml

```diff
@@ -929,6 +929,21 @@ repositories:
       url: https://github.com/g/grizzly.git
       version: indigo-devel
     status: maintained
+  heatmap:
+    doc:
+      type: git
+      url: https://github.com/eybee/heatmap.git
+      version: jade-devel
+    release:
+      tags:
+        release: release/jade/{package}/{version}
+      url: https://github.com/eybee/heatmap-release.git
```

Comment:
This is not a release repository. You might want to look into the first bloom tutorial (http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) how to release ROS packages.